deploy/run_with_summon.sh

```diff
@@ -4,15 +4,17 @@ set -xeuo pipefail
 . utils.sh
 
 # Clean up when script completes and fails
-function finish {
+finish() {
   # There is a TRAP in test_in_docker.sh to account for Docker deployments so we do not need to add another one here
   # Stop the running processes
   if [[ $RUN_IN_DOCKER = false && $DEV = false ]]; then
     announce 'Wrapping up and removing environment'
-    ./stop
-    cd ./kubernetes-conjur-deploy-$UNIQUE_TEST_ID && ./stop
-    # Remove the deploy directory
-    rm -rf "../kubernetes-conjur-deploy-$UNIQUE_TEST_ID"
+    pushd ../../ > /dev/null
```

Comment:
will this function run also after successful runs? if so, this `pushd` may be wrong

Reply:
fixed